packages/node/src/integrations/local-variables/worker.ts

```diff
@@ -118,7 +118,9 @@ async function handlePaused(
   // We write the local variables to a property on the error object. These can be read by the integration as the error
   // event pass through the SDK event pipeline
   await session.post('Runtime.callFunctionOn', {
-    functionDeclaration: `function() { this.${LOCAL_VARIABLES_KEY} = ${JSON.stringify(frames)}; }`,
+    functionDeclaration: `function() { this.${LOCAL_VARIABLES_KEY} = this.${LOCAL_VARIABLES_KEY} || ${JSON.stringify(
+      frames,
+    )}; }`,
```

Comment:
I love how simple this change is now after the redesign.
![image](https://github.com/user-attachments/assets/4bb890b0-78e0-4955-a6da-00ad22f60dec)

Reply:
Shame it doesn't appear to work now I'm adding a test for it!

Reply:
Ah no, I forgot the recent changes only fix this issue in Node v20+ 🤦🏻‍♂️